packages/edit-post/src/components/layout/index.js

```diff
@@ -266,6 +266,7 @@ function Layout() {
 				<KeyboardShortcutHelpModal />
 				<WelcomeGuide />
 				<Popover.Slot />
+				<Popover.Slot name="block-toolbar" />
```

Comment:
I believe this slot was not rendered her for a valid reason. Otherwise, it just duplicated the slot that is above it. cc @ellatrix

Reply:
@youknowriad Hmm! Interesting.. I wasn't aware of that !

Reply:
This needs to be rendered in the content scroll container, because when you scroll while the pointer is over the toolbar, the content still needs to scroll.

Reply:
I should have commented here 😅
There might be other reasons I'm forgetting.